apps/devhub/templates/devhub/includes/macros.html

```diff
@@ -76,3 +76,11 @@
     {{ form.categories.errors }}
   </div>
 {% endmacro %}
+
+{% macro url_if_listed(addon_or_version, text) %}
```

Comment:
also, the name `xxx_if_listed` doesn't say what happens when it's not listed, maybe that's a problem too...